File: Java/arrays/sum_2_arrays.java
package br.com.geekuniversity.secao08;

import java.util.Scanner;

public class Exercicio02 {

	public static void main(String[] args) {
		//Declaração
		int vetor1[] = new int [10];
		int vetor2[] = new int [10];
		int soma[] = new int [10];
		Scanner entrada = new Scanner (System.in);
		
		//Entrada
		for (int i = 0; i < 10; i++) {
			System.out.println("Digite um valor para o vetor 1: ");
			vetor1[i] = entrada.nextInt();
			System.out.println("Digite um valor para o vetor 2: ");
			vetor2[i] = entrada.nextInt();
			//Processamento
			soma[i] = vetor1[i] + vetor2[i];
		}
		
		//Saída
		for (int i = 0; i < 10; i++) {
			System.out.printf("%d ", soma[i]);
		}
		entrada.close();
	}
}

File: C/arrays/sum_2_arrays.c
#include <stdio.h>

int main() {
	//Declaração
	int vetor1[5], vetor2[5], soma[5];

	//Entrada
	for (int i = 0; i < 5; i++) {
		printf ("Digite um valor para o vetor 1: ");
		fflush (stdout);
		scanf ("%d", &vetor1[i]);
		soma[i] = vetor1[i];
	}
	for (int i = 0; i < 5; i++) {
		printf ("Digite um valor para o vetor 2: ");
		fflush (stdout);
		scanf ("%d", &vetor2[i]);
		//Processamento
		soma[i] = vetor1[i] + vetor2[i];
	}

	//Saída
	for (int i = 0; i < 5; i++) {
		printf ("%d ", soma[i]);
	}
}
